README.md

```diff
@@ -1 +1,102 @@
-# atlas_starter_nodejs
\ No newline at end of file
+# MongoDB Atlas Node.js Sample Project
+
+This repository contains an example application that connects to MongoDB
+Atlas using the Node.js MongoDB driver. You can use this application as
+a reference for when you build your Node.js application.
+
+## Prerequisites
+
+To build and run this project, you need Node.js version 10 or later and
+a compatible version of NPM (Node Package Manager) installed. For 
+information on how to check your version of Node and NPM, see the [NPMJS
+page on downloading and installing Node.js and
+npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+## Getting Started
+
+The following instructions explain how to get your project connected to your 
+instance of MongoDB Atlas.
+
+### 1. Download the Repository
+
+To get started with this sample project, download this repository to your
+programming environment. You can clone this project using Git version control:
+
+```
+git clone https://github.com/mongodb-university/atlas_starter_nodejs.git
+```
+
+Or you can download the ZIP archive using your browser. If you download
+this project as a ZIP archive, 
+[unzip the archive](https://www.wikihow.com/Unzip-a-File) before proceeding.
+
+### 2. Install Depedencies
+
+Navigate to the directory containing the project in your shell or open it
+in your preferred IDE.
+
+Then, set up NPM for your project by running the following command which
+creates a file called `package.json`:
+
+```shell
+npm init -y
+```
+
+Next, add the Node.js MongoDB driver as a dependency. Use the following
+command to instruct NPM to download and install the `mongodb` package.
+
+```shell
+npm install mongodb
+```
+
+This command downloads and saves the dependencies required in a directory
+called `node_modules` in your project directory and records dependency
+information in the `package.json` file you generated in the previous step.
+
+At this point, you should have appropriate versions of Node.js and NPM 
+installed as well as a project directory that contains the dependencies you 
+need to use the Node.js MongoDB driver.
```

Comment:
Are we adverse to including a package.json in this repo?  Would remove a few steps from the user's process.

Reply:
I think it's better for them to generate it so that they know how to start a project from scratch and it also serves to verify that thy have npm installed. I could see how you might provide all the files for them if this was an application scaffold, but the current structure doesn't seem to resemble a proper scaffold. 
Personally, I think the commands contained in this starter project are fairly static and probably require much less maintenance than updating the package.json every time a new version of the Node driver is released, assuming one of the objectives is to have them work with the latest drivers.

Reply:
`npm intstall` would also confirm they've got NPM installed.

We wouldn't need to update the package.json every time a new driver is released.  [Only when a major version is released](https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/).  And if a major version is released without pinning, we run the risk of not noticing this repo no longer functions.  

Including a package.json is more standard than not in example apps.  With all of this in place, plus the removal of a step from the onboarding process, I don't see the down side to including a package.json.  I'd like to push for this one.

Reply:
To use a Linux analogy, I think the "Arch Linux distro" approach is more helpful to learn to create the app than "Debian Linux distro", but see merits in both, understand your concern, and will update.